src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java

```diff
@@ -115,6 +118,11 @@ public CompletableFuture<Result<Item>> updateItem(Item item) {
       write(updatedItemRepresentation, LAST_CHECK_IN, lastCheckIn.toJson());
     }
 
+    if(item.isDCBItem()){
+      return circulationItemRepository.updateItem(item.getItemId(), updatedItemRepresentation)
+        .thenApply(noContentRecordInterpreter(item)::flatMap)
+        .thenCompose(x -> ofAsync(() -> item));
```

Comment:
It looks like nothing is actually happening here, we're just mapping the result. Is thenCompose really justified?